packages/keyboardfocus/src/useKeyboardFocus.js

```diff
@@ -39,6 +54,8 @@ export function useKeyboardFocus() {
     };
   };
 
+  console.log(keyboardFocused);
```

Comment:
Remove this please

Reply:
Not sure why the husky pre-commit didn't pick this up? Removed.